### ISSUE TITLE
Try to run systemd-analyze with the help of container support

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/meta-sca#299]
**Is your feature request related to a problem? Please describe.**
Within systemd a tool exists that can verify any systemd-unit. Unfortunately this does only run when being cross-compiled in the actual system, which is kind of senseless, as we need the info before starting the image.
Within poky-layer sometimes a qemu-wrapper is used for starting a single binary which has been already cross-compiled. So here's the deal: try to start the cross-compiled systemd-binary via qemu-wrapper with the resulting rootfs and analyze the results

**Describe a tool that might help here**
See python-implementation of PGO from poky...

Edit 2019/09/27

Unfortunately it seems that qemu-wrapper didn't help, as the tool can be run but doesn't find it's dependencies + a lot of unrelated error on run - for me it's better to retest the idea when running things in a container --> conclusion: move to container support milestone

**Comment by priv-kweihmann** [comment]
This one is really complicated.
With the help of qemu-wrapped (chrooted) I was able to run systemd-analyze as a tool, which was then complaining about that cgroup of systemd can't be determined and no dbus-comm is possible.

So there might be a lot of work to do here, to get things going.
BTW systemd is crappy by design, if the configuration can only be checked if the compilation unit is already running...

**Comment by priv-kweihmann** [comment]
I recently put some effort into this, to just get an idea how to solve this - without any results in the end.
Currently I don't see any way to validate a unit/system setup of systemd (with the help of systemd) without booting a system with systemd - so this COULD be solved by using containers or somethings similar, but seems to be the complete overkill for some task that should be that simple (in theory).

I closing this one and maybe open up a new issue if there will be another option available
